apps/mull-ui/src/app/components/chat-input/chat-input.scss

```diff
@@ -44,6 +44,11 @@
   }
 }
 
+.image-size {
```

Comment:
💡Same as before, it's a good idea to namespace your classes since otherwise they might clash with other places and cause a lot of confusion.
```suggestion
.chat-input-image-size {
```